### ISSUE TITLE
Update radcli gem with latest adcli code

### ISSUE THREAD

**Issue by wiad** [theforeman/smart_proxy_realm_ad_plugin#23]
New Microsoft patches for Active Directory will prohibit the use of unsigned and simple binds. The use of GSSAPI, which radcli/adcli uses, is registered as an unsigned auth request. There is a discussion about this regarding SSSD which uses adcli for renewing machine account passwords. 
https://www.mail-archive.com/search?l=sssd-users@lists.fedorahosted.org&q=subject:%22%5C%5BSSSD%5C-users%5C%5D+Re%5C%3A+How+do+new+LDAP+security+recommendations+from+MS+affect+sssd+clients%5C%3F%22&o=oldest

Long story short, using SPNEGO instead of GSSAPI fixes this and adcli is patched to try SPNEGO since a couple of weeks back:
https://gitlab.freedesktop.org/realmd/adcli/commit/a6f795ba3d6048b32d7863468688bf7f42b2cafd

So it would be nice if the `rubygem-radcli` was rebuilt with the latest adcli code. There is also other stuff from radcli which would be nice to get (for example #20).

**Comment by martencassel** [comment]
Hi @wiad could like to be added as a maintiner in  for radcli and smart_proxy_realm_ad_plugin at rubygems.org ?

**Comment by wiad** [comment]
I am actually pretty rubbish at ruby and much prefer to install and use pre-packaged (rpm) rubygems, so I don't feel really comfortable with it. I will check with a colleague of mine who is much better versed in ruby.

**Comment by martencassel** [comment]
https://github.com/martencassel/radcli/pull/7

**Comment by martencassel** [comment]
I have updated radcli on github (https://github.com/martencassel/radcli) to use https://github.com/freedesktop/realmd-adcli/tree/0.9.0

**Comment by evgeni** [comment]
This happened in nightly now -- would y'all mind testing it if possible?

**Comment by martencassel** [comment]
Where is nightly on github of what ? :-) 
How would i test it ?

**Comment by evgeni** [comment]
Yes.

You should be able to use the rpm directly on any install: https://yum.theforeman.org/plugins/nightly/el8/x86_64/rubygem-radcli-1.1.0-1.el8.x86_64.rpm

**Comment by martencassel** [comment]
hi @evgeni see the latest PR
this project needs a stable test setup,
so before that is done i can start fixing these issues that been here for ages.

**Comment by martencassel** [comment]
> Yes.
> 
> You should also be able to just use the rpm directly on any install: https://yum.theforeman.org/plugins/nightly/el8/x86_64/rubygem-radcli-1.1.0-1.el8.x86_64.rpm
Couldn't find one for debian nightly  ? Do you build that ?

**Comment by martencassel** [comment]
https://github.com/martencassel/radcli/pulls
